intlogger.go

```diff
@@ -353,20 +377,36 @@ func (l *intLogger) logPlain(t time.Time, name string, level Level, msg string,
 				}
 			}
 
-			var key string
-
+			// Convert the field key to a string.
 			switch st := args[i].(type) {
 			case string:
 				key = st
 			default:
 				key = fmt.Sprintf("%s", st)
 			}
 
+			// Optionally apply the ANSI "faint" and "bold"
+			// SGR values to the key.
+			if l.fieldColor != ColorOff {
+				key = faintBoldColor.Sprint(key)
+			}
+
+			// Values may contain multiple lines, and that format
+			// is preserved, with each line prefixed with a "  | "
+			// to show it's part of a collection of lines.
+			//
+			// Values may also need quoting, if not all the runes
+			// in the value string are "normal", like if they
+			// contain ANSI escape sequences.
 			if strings.Contains(val, "\n") {
 				l.writer.WriteString("\n  ")
 				l.writer.WriteString(key)
 				l.writer.WriteString("=\n")
```

Comment:
Did you opt to leave the = as "normal" and not faint then?

Reply:
Oops, that was actually unintentional. Lemme see if I can get that resolved before merging!

I'll be sure to squash 👍

Reply:
Ah, I also just realized I _missed_ the `!raw && needsQuoting(val)` case as well. Need to fix that too.

Reply:
Fixed with https://github.com/hashicorp/go-hclog/pull/118/commits/fd264152857468feff6a784c047e6585bebdd664